minio/minio
# info: Populate pool/set/disk indexes for offline disks

## Community Contribution License
All community contributions in this pull request are licensed to the project maintainers
under the terms of the [Apache 2 license](https://www.apache.org/licenses/LICENSE-2.0). 
By creating this pull request I represent that I have the right to license the 
contributions to the project maintainers under the Apache 2 license.

## Description
This can be calculated from the disk layout and some external applications
would like to know the location of the offline disks.

## Motivation and Context


## How to test this PR?
Start a distributed setups of 4 nodes and kill one node; then run
`mc admin info <alias> --json` and check for pool/set/disk index
of the offline disks.

## Types of changes

## Checklist:
